examples/httpscaledobject.yaml

```diff
@@ -6,3 +6,5 @@ spec:
     app_name: xkcd
     app_image: arschles/xkcd
     port: 8080
+    min_replicas: 5
+    max_replicas: 10
```

Comment:
How about this?
```suggestion
    replicas:
      min: 5
      max: 10
```

Reply:
I like it, will update the API to make this the default

Reply:
Resolved in last commit